dist/templates/ovn-ipsec.yaml.j2

```diff
@@ -258,8 +258,8 @@ spec:
         # To check that network setup is complete
         - mountPath: /etc/cni/net.d
           name: host-cni-netd
-        - mountPath: /var/run/openvswitch
-          name: host-var-run-ovs
+        - mountPath: /var/run
```

Comment:
what else are you consuming from run? why not list it specifically instead of the whole dir?